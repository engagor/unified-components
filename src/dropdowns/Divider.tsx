import React from "react";
import "@clarabridge/unified-ui-tailwind/dist/utilities.css";

export default function Divider() {
    return <hr className="border-0 border-b-1 border-solid border-primary-color-600 mb-8 mt-8" />;
};
