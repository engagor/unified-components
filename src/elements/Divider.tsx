import React, { ReactElement } from 'react';

export type DividerProps = {
    color?: string;
    className?: string;
};

export default function Divider(props: DividerProps): ReactElement {
    return <hr className={`border-0 border-b-1 border-solid ${props.color ?? 'border-charcoal-color-200'} ${props.className ?? ''}`} />
}
