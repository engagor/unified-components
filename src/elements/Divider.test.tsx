import React from 'react';
import renderer from 'react-test-renderer';
import Divider from './Divider';

it('renders correctly', async () => {
    const element = document.createElement('div');

    const divider = await renderer
        .create(<Divider />)
        .toJSON();

    await expect(divider).toMatchSnapshot();
    expect(element).toMatchSnapshot();
});
