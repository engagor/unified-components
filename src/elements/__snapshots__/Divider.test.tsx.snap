// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<hr
  className="border-0 border-b-1 border-solid border-charcoal-color-200 "
/>
`;

exports[`renders correctly 2`] = `<div />`;
